__tests__/articles.test.js

```diff
@@ -90,3 +90,114 @@ describe("GET/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET/api/articles/:article_id/comments", () => {
+  test("responds with a status of 200, returns an array of comments with the corrosponding article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("responds with a status of 200, returns an array of comments with the corrosponding article_id sorted by the newest comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("responds with a 404 error when passed a article id which doesnt exist", () => {
+    return request(app)
+      .get("/api/articles/464646/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("Bad request");
+      });
+  });
+
+  test("responds with a 400 error when passed an article id formatted incorrectly", () => {
+    return request(app)
+      .get("/api/articles/incorrectformat/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("responds status 201 with an object containing the new comment", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "testing post",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const {comment} = body
+        expect(comment).toEqual({
```

Comment:
I would recommend swapping out `.toEqual` for `.toMatchObject` - as the latter will still pass if there are additional properties in the object , this makes your test suite open to expansion as if down the line your comment object body has additional properties added to it your current tests will still pass